docs/reference/data-streams/logs.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[logs-data-stream]]
+== Logs data stream
+
+preview::[Logs data streams and the logs index mode are in tech preview and may be changed or removed in the future. Don't use logs data streams or logs index mode in production.]
+
+A logs data stream is a data stream type that stores log data more efficiently.
+
+In benchmarks, log data stored in a logs data stream used ~2.5 times less disk space than a regular data
+stream. The exact impact will vary depending on your data set.
+
+The following features are enabled in a logs data stream:
+
+* <<synthetic-source,Synthetic source>>, which omits storing the `_source` field. When the document source is requested, it is synthesized from document fields upon retrieval.
+
+* Index sorting. This yields a lower storage footprint. By default indices are sorted by `host.name` and `@timestamp` fields at index time, which can be overwritten.
+
+* More space efficient compression for fields with <<doc-values,`doc_values`>> enabled.
+
+The new logs data stream should be used for all data streams that store logs, because of the storage benefits that comes with it.
+
+[discrete]
+[[how-to-use-logsds]]
+=== Create a logs data stream
+
+To create a logs data stream, set your index template  `index.mode` to `logs`:
+
+[source,console]
+----
+PUT _index_template/my-index-template
+{
+  "index_patterns": ["logs-*"],
+  "data_stream": { },
+  "template": {
+     "settings": {
+        "index.mode": "logs" <1>
+     }
+  },
+  "priority": 101 <2>
+}
+----
+// TEST
+
+<1> The index mode setting.
+<2> Defining a priority to avoid other index templates with overlapping index patterns potentially overriding the settings. Elasticserch by default ships with an index template with a `logs-*-*` pattern with a priority of 100. Defining a priority higher than 100 ensures that this index template gets selected over the default index template for `logs-*-*` pattern. See the <<avoid-index-pattern-collisions,avoid index pattern collision section>> for more information.
+
+After the index template is created, you can start indexing data and <<use-a-data-stream,using the data stream>>.
```

Comment:
consider this small tweak so it's accurate until you follow up with migrating existing data streams

```suggestion
After the index template is created, new indices that use the template will be configured as a logs data stream. You can start indexing data and <<use-a-data-stream,using the data stream>>.
```